fork-off/src/config.rs

```diff
@@ -47,7 +42,54 @@ pub struct Config {
         multiple_occurrences = true,
         takes_value = true,
         value_delimiter = ',',
-        default_value = "Aura,Aleph,Balances,Sudo,Staking,Session,Elections,System.Account"
+        default_value = "Aura,Aleph,Sudo,Staking,Session,Elections"
     )]
     pub storage_keep_state: Vec<StoragePath>,
+
+    #[clap(long)]
+    pub accounts_path: Option<String>,
+
+    #[clap(
+        long,
+        parse(try_from_str = parse_balances),
+        value_delimiter = ',',
+        multiple_occurrences(true))
+    ]
+    pub balances: Option<Vec<(AccountId, Balance)>>,
+}
+
+fn parse_balances(s: &str) -> Result<(AccountId, Balance), Box<dyn Error + Send + Sync + 'static>> {
+    let sep_pos = s.find('=').ok_or("Invalid ACCOUNT=BALANCE: no `=` found")?;
+
+    let account_raw: String = s[..sep_pos].parse()?;
+    let account = AccountId::new(&account_raw);
```

Comment:
This accepts hex-based `AccountId`, is it not the case for end-users who mostly use the `SS58` format? Just wondering what would be most convenient for them to use.

Reply:
uhm, I was struggling with encoding account to the correct hash for a long time and for now it is the only way I can do this;

you can always use `subkey` to see this hex-based public key

Reply:
You could try:

- trim 0x
- let bytes = hex::decode_to_slice(trimmed)
- AccountId::new(bytes)

There is also : [AccountId::from_string](https://github.com/paritytech/substrate/blob/polkadot-v0.9.19/primitives/core/src/crypto.rs#L420)